### ISSUE TITLE
<title>

### ISSUE THREAD

**Issue by MEZ0SFER** [ValkyrienSkies/Eureka#426]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.20​

### Eureka version

1201-1.5.1-beta.3

### VS2 version

1.5.1-beta.3

### Mod Loader

Fabric

### Issue description

Hi, I am Muzaffer and I have a weird issue with the ships that I build. When I assemble my ship one of the 6 sides of the entity (north, south, east, west, up or down) becomes invisible. Not a great issue but it's really annoying to see through your ship. I am using modpack called Prominence Hasturian Era 1.20.1 but even I tried without it, result is same. Could you find a solution for me pls?

Thank you

### Issue reproduction

I don't know why it's happening but it just happens every time and a side becomes invisible randomly.

**Comment by thetheaplant** [comment]
Please do at least minimal research before posting issues, even just googling the issue. If you had done that, you would have found that the solution is disabling block face culling in your performance settings.

**Comment by Tyrannicalized** [comment]
Yeah, no. Disabling that just turns the entire ship invisible instead of just some block faces for me.

**Comment by Tyrannicalized** [comment]
It's only after tinkering with block faces, as you mentioned, that i found out you need to also refresh or reload your build, or wait a few and it should come back.
